osu.Game/Utils/SupportedExtensions.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Utils
+{
+    public static class SupportedExtensions
+    {
+        public static readonly string[] VIDEO_EXTENSIONS = [@".mp4", @".mov", @".avi", @".flv", @".mpg", @".wmv", @".m4v"];
+        public static readonly string[] AUDIO_EXTENSIONS = [@".mp3", @".ogg", @".wav"];
+        public static readonly string[] IMAGE_EXTENSIONS = [@".jpg", @".jpeg", @".png"];
+
+        public static readonly string[] ALL_EXTENSIONS =
+        [
+            ..VIDEO_EXTENSIONS,
+            ..AUDIO_EXTENSIONS,
+            ..IMAGE_EXTENSIONS
+        ];
```

Comment:
Primer for what this is: https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-12#collection-expressions

I find this to be a compelling usage of the syntax because any other way to write it in C# is veeeeeeeeeery clunky, but I will go the clunky route if usage of this syntax is considered not desired.

Reply:
I personally think it's fine as it shortens code while not looking extremely odd.

Reply:
I think I'm okay with this one.